src/ml_gym/frontend/dashboard/src/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
```

Comment:
Codacy has a fix for the issue: [Replace `'./App'` with `"./App"`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
import App from "./App";
```